### ISSUE TITLE
PPLX model selection

### ISSUE THREAD

**Issue by bm777** [bm777/hask#5]
PPLX model selection

**Comment by bm777** [comment]
models: 

```js
const [models, setModels] = useState(["pplx-7b-online", "pplx-70b-online"]);
```

---------
Model selection in useState, works. Even after retrieving from localStorage.
But, other providers are locked!

**Comment by bm777** [comment]
solved in [b7cdfe7](https://github.com/bm777/hask/commit/b7cdfe7b3524ca3d160d6cbd515ff079c378c2ca)
